pkg/apis/portworx/v1beta1/types.go

```diff
@@ -193,5 +193,41 @@ type VolumePlacementRule struct {
 	// Type is the type of the affinity rule
 	Type AffinityRuleType `json:"type,omitempty"`
 	// MatchExpressions is a list of label selector requirements. The requirements are ANDed.
-	MatchExpressions []*meta.LabelSelectorRequirement `json:"matchExpressions,omitempty"`
+	MatchExpressions []*LabelSelectorRequirement `json:"matchExpressions,omitempty"`
+}
+
+// LabelSelectorOperator is the set of operators that can be used in a selector requirement.
+type LabelSelectorOperator string
+
+const (
+	// LabelSelectorOpIn is operator where the key must have one of the values
+	LabelSelectorOpIn LabelSelectorOperator = "In"
+	// LabelSelectorOpNotIn is operator where the key must not have any of the values
+	LabelSelectorOpNotIn LabelSelectorOperator = "NotIn"
+	// LabelSelectorOpExists is operator where the key must exist
+	LabelSelectorOpExists LabelSelectorOperator = "Exists"
+	// LabelSelectorOpDoesNotExist is operator where the key must not exist
+	LabelSelectorOpDoesNotExist LabelSelectorOperator = "DoesNotExist"
+	// LabelSelectorOpGt is operator where the key must be greater than the values
+	LabelSelectorOpGt LabelSelectorOperator = "Gt"
```

Comment:
nit: If other operators are verbose why not keep these verbose as well - LabelSelectorOpGreaterThan = "GreaterThan"?

Reply:
Gt and Lt is what Kubernetes already calls it 

From https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity,

```
The new node affinity syntax supports the following operators: In, NotIn, Exists, DoesNotExist, Gt, Lt. You can use NotIn and DoesNotExist to achieve node anti-affinity behavior, or use node taints to repel pods from specific nodes.
```

So keeping it consistent.